### PR TITLE
Feature/wallet bonding with fees rebase

### PR DESCRIPTION
**Pull request by fmtabbara**
# Description

Display simulated gas fees for bonding/unbonding mixnodes and gateways with locked or unlocked coins

## Closes:
https://github.com/nymtech/nym/issues/2379
https://github.com/nymtech/nym/issues/2378
https://github.com/nymtech/nym/issues/2377
https://github.com/nymtech/nym/issues/2376

### CODE REVIEW

File: nym-wallet/src/components/Modals/LoadingModal.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Box, CircularProgress, Modal, Stack, Typography } from '@mui/material';
+
+const modalStyle = {
+  position: 'absolute' as 'absolute',
```
Comment (doums) [inline nym-wallet/src/components/Modals/LoadingModal.tsx]: are you sure about this cast `'absolute' as 'absolute'` ?

File: ts-packages/types/src/types/global.ts [hunk 2]
```diff
@@ -1,13 +1,10 @@
 import { MixNode, MajorCurrencyAmount } from './rust';
 
-export enum EnumNodeType {
-  mixnode = 'mixnode',
-  gateway = 'gateway',
-}
+export type TNodeType = 'mixnode' | 'gateway';
```
Comment (doums) [inline ts-packages/types/src/types/global.ts:3]: :godmode:

**Comment by doums** [comment]
LGTM

File: nym-wallet/src/components/Modals/LoadingModal.tsx (hunk 1 above)
Comment (fmtabbara) [reply nym-wallet/src/components/Modals/LoadingModal.tsx]: I think I borrowed this style object from some code Mark wrote. So I'm not sure why the cast is there but my guess is to satisfy something in the MUI sx prop. will see if we can remove the cast without any issues

Comment (fmtabbara) [reply nym-wallet/src/components/Modals/LoadingModal.tsx]: I managed to remove it :100:

**Pull request merged**